spec/text-editor-presenter-spec.coffee

```diff
@@ -2227,6 +2099,407 @@ describe "TextEditorPresenter", ->
         expectStateUpdate presenter, -> presenter.setFocused(false)
         expect(presenter.getState().focused).toBe false
 
+    describe ".gutters", ->
+      describe ".scrollHeight", ->
```

Comment:
Oh it seems like it *does* have an entry in the `gutters` list. Why have that plus another top level `.lineNumberGutter` key?